packages/python/src/armonik/client/partitions.py

```diff
@@ -47,19 +74,25 @@ def list_partitions(self, partition_filter: Filter | None = None, page: int = 0,
         request = ListPartitionsRequest(
             page=page,
             page_size=page_size,
-            filters=cast(rawFilters, partition_filter.to_disjunction().to_message()) if partition_filter else None,
-            sort=ListPartitionsRequest.Sort(field=cast(PartitionField, sort_field.field), direction=sort_direction),
+            filters=cast(rawFilters, partition_filter.to_disjunction().to_message())
+            if partition_filter
+            else None,
```

Comment:
I don't really like this one. I prefer it as a one liner since you know what it is more easily. As that qe could think it is multiple parameters

Reply:
The line will be longer than 88. The linter does not have any other possibility.